docs/Advanced-Usage/FireAxe-Partitioning-onto-Multiple-FPGAs/Running-Fast-Mode-Simulations.rst

```diff
@@ -0,0 +1,137 @@
+.. _FIREAXE-FAST-MODE:
+
+Running Fast Mode Simulations
+===================================
+
+In this section, we provide a step-by-step tutorial on how to partition a
+*RocketTile* out from the *SoC* and run fast-mode simulations using 2 FPGAs on EC2 F1.
+This assumes that you have completed the :ref:`AWS F1 Getting Started Guide<AWS-F1-Getting-Started-Guide>` and that you are familiar with
+the FireSim workflow.
+
+1. Building Partitioned Sims : Setting up FireAxe Target configs
```

Comment:
Nit: Everywhere you use `WORD : WORD`. To keep this consistent with the rest of the docs remove the 1st space `WORD: WORD`.